web3Admin.js

```diff
@@ -6,6 +6,13 @@ module.exports = {
             property: 'admin',
             methods:
             [
+		new web3._extend.Method({
+                    name: 'addBlock',
+                    call: 'admin_addBlock',
```

Comment:
Ah, I'm sorry, I think that was just part of your PR that I imported. I will delete it.